.github/workflows/coccicheck.yaml

```
@@ -0,0 +1,12 @@
+name: BPF
+
+on:
+  - pull_request
+
+jobs:
+    coccicheck:
+        name: coccicheck
+        runs-on: ubuntu-18.04
+        steps:
+            - uses: actions/checkout@v1
```

Similar to @qmonnet's comment above, we could use `actions/checkout@v2` like in `go-mod.yaml`. Also see https://github.com/actions/checkout#whats-new

```suggestion
            - uses: actions/checkout@v2
```

---

Thanks for including the link!